official/recommendation/ncf_estimator_benchmark.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Executes Keras benchmarks and accuracy tests."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import time
+
+from absl import flags
+import tensorflow as tf  # pylint: disable=g-bad-import-order
+
+from official.recommendation import benchmark_base
+from official.recommendation import ncf_estimator_main
+
+FLAGS = flags.FLAGS
+
+
+class KerasNCFRealData(benchmark_base.KerasNCFBenchmarkBase):
```

Comment:
Class name should be:
NCFEstimatorReal  or EstimatorNCFReal .   RealData is also ok if desired.